agent/session_endpoint_test.go

```diff
@@ -385,6 +385,9 @@ func TestSessionTTLRenew(t *testing.T) {
 	if err != nil {
 		t.Fatalf("err: %v", err)
 	}
+	if obj == nil {
```

Comment:
The result from SessionGet won't be nil, there's a check for that before it returns: https://github.com/hashicorp/consul/blob/master/agent/session_endpoint.go#L205

If you want to make sure that's not empty you could do `len(obj) == 0`.